lib/tasks/valid_test_data_generator.rb

```diff
@@ -108,7 +108,37 @@ def weighted_choice(selection:, odds:)
   class AmbitionSpecificPopulater < LeadProviderPopulater
     class << self
       def call(name:, total_schools: 3, participants_per_school: 3000)
-        new(name: name).call(total_schools: total_schools, participants_per_school: participants_per_school)
+        generator = new(name: name)
+        generator.remove_old_data(created_at: "2021-08-18 13:43".."2021-08-18 13:49")
```

Comment:
This `created_at` range seems very specific for something going into the general rake task.

Can we maybe put it in a constant, like "FIRST_DATA_SEED_TIME" or something?

Reply:
Yes. Whole rake task is very specific too though. It can be run again, but will always only add a delta to the previous run. I'll call it FIRST_AMBITION_SEED_DATA_TIME, since that was when their pass happened rather than all of them.